index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Santiago's Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    
+</head>
+<body>
+    <section class="toolbar">
+        <div>
+            <a href="#"><img src="images/logo.png"></a>
+        </div>
+        <nav class="navigation">
+            <a href="#">Portfolio</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+        </nav>
+        <div class="menu">
+            <a href="#"><img src="images/Union.png"></a>
+        </div>
```

Comment:
Good job adding a hamburger 🍔 icon but let's improve this. The hamburger button should be on the right and have a `nav` tag.

* The `nav` tag is for semantics reasons, and the right position for convention and user experience for mobile devices. 📱